addons/0-byte-files.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2021, https://www.hanssonit.se/
+true
+SCRIPT_NAME="Check for 0-Byte files"
+# shellcheck source=lib.sh
+source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+filesystems[0]="$NCDATA"
+cd "$NCDATA"
+
+emSub="Warning: Nextcloud contains 0-Byte files!"
+
+# Info
+msg_box "We will now scan $NCDATA for files that are 0-Byte.
+
+A 0-Byte file means that it's empty and probably corrupted/not usable. If you see files that are of importance in this list, you should report it immediately to $ISSUES.
+
+The scan may take very long time depending on the speed of your disks, and the amount of files."
+
+countdown "The scan starts in 3 seconds..." "3"
+
+print_text_in_color "$ICyan" "Scan in progress, please be patient..."
+
+# Function
+for fs in "${filesystems[@]}"
+do
+    while IFS= read -d '' -r
+    do
+        arr+=( "$REPLY" )
+    done < <(find "$fs" -mindepth 3 -size 0 -print0)
+done
+
+
+# TODO
+# Remove known 0-byte files
+# Everything in appdata isn't important
+# NEWARR="$(echo ${arr[*]} | sed s/'appdata'//)"
+
+# Notify!
+if [ -n "${arr[*]}" ]
+then
+    send_mail "$emSub" "${arr[*]}"
```

Comment:
@enoch85 this won't work without having smtp-mail set up. I'd recommend to test if it is installed by simply doing something like this: 
https://github.com/nextcloud/vm/blob/5b04abac6e54e0f6383f96001615af1c13ac655a/not-supported/daily-backup-wizard.sh#L154-L158

Reply:
At the same time it won't give any output if not installed, so if they have it it works, if not, then too bad. :)